### ISSUE TITLE
Incorrect orientation of the splashscreen, in portrait view in android

### ISSUE THREAD

**Issue by gabimem** [godotengine/godot#350]
Hi, 
The error occurs in an application in portrait view. 

![image](https://cloud.githubusercontent.com/assets/5366368/2825364/a11e84b2-cf4a-11e3-85a3-679f68ebdb52.png)

The settings are: "config / display / orientation / portrait" in the project settings, and "screen / orientation / portrait" in the export settings for android. 

The display screen shows the logo incorrectly (horizontally), in portrait view. 

This apparently happens because when the 2d context is created, the game is first resized to the dimensions of the landscape view, and then resized to the dimensions of the vertical view. 

Apparently, the context is created and resized before reading the settings. 

This is what the logcat shows:
```
I/System.out( 7146): ** GODOT ACTIVITY CREATED HERE ***
I/godot   ( 7146): **INIT EVENT! - 0xad20
I/godot   ( 7146): ***************** HELLO FROM JNI!!!!!!!!
I/godot   ( 7146): *******CLASS FOUND!!!
I/godot   ( 7146): STEP2, 0x4068a850
I/godot   ( 7146): STEP3 1080600840
I/godot   ( 7146): STEP4, 0x4068c020
I/godot   ( 7146): STEP4.5, 0x4068c020
I/godot   ( 7146): STEP5
I/godot   ( 7146): STEP6
I/godot   ( 7146): *******GOT METHOD _file_open ok!!
I/godot   ( 7146): *******GOT METHOD _file_get_size ok!!
I/godot   ( 7146): *******GOT METHOD _file_tell ok!!
I/godot   ( 7146): *******GOT METHOD _file_eof ok!!
I/godot   ( 7146): *******GOT METHOD _file_seek ok!!
I/godot   ( 7146): *******GOT METHOD _file_read ok!!
I/godot   ( 7146): *******GOT METHOD _file_close ok!!
I/godot   ( 7146): STEP7
I/godot   ( 7146): STEP8
I/godot   ( 7146): *******GOT METHOD _dir_open ok!!
I/godot   ( 7146): *******GOT METHOD _dir_next ok!!
I/godot   ( 7146): *******GOT METHOD _dir_close ok!!
I/godot   ( 7146): starting to attempt get methods
I/godot   ( 7146): *******GOT METHOD _init_audio ok!!
I/godot   ( 7146): *******GOT METHOD _write_buffer ok!!
I/godot   ( 7146): *******GOT METHOD _quit ok!!
I/godot   ( 7146): *******GOT METHOD _pause ok!!
I/godot   ( 7146): [1;31;40mERROR: create: [1;37;40mCondition ' !create_func ' is true. returned: 0
I/godot   ( 7146): [0;31;40m   At: core/os/mutex.cpp:39.[0;0;37m
I/godot   ( 7146): test construction 5
I/godot   ( 7146): running from dir /
I/godot   ( 7146): **SETUP
I/godot   ( 7146): has res dir: 
I/godot   ( 7146): exists engine cfg? 0
I/godot   ( 7146): loaded engine.cfg
I/godot   ( 7146): *****SETUP OK
W/dalvikvm( 7146): VFY: unable to resolve virtual method 402: Lcom/android/godot/GodotView;.setPreserveEGLContextOnPause (Z)V
I/godot   ( 7146): **START
W/GodotView( 7146): creating OpenGL ES 1.1 context :
I/godot   ( 7146): ^_^_^_^_^ newcontext 2279704
I/godot   ( 7146): ^_^_^_^_^ resize 2279704, 480, 320
I/godot   ( 7146): mod count: 0
I/godot   ( 7146): OpenSL Init OK!
I/godot   ( 7146): [1;31;40mERROR: set_custom_property_info: [1;37;40mCondition ' !props.has(p_prop) ' is true.
I/godot   ( 7146): [0;31;40m   At: core/globals.cpp:1318.[0;0;37m
I/godot   ( 7146): [1;31;40mERROR: set_stylebox: [1;37;40mCondition ' p_style.is_null() ' is true.
I/godot   ( 7146): [0;31;40m   At: scene/resources/theme.cpp:267.[0;0;37m
I/godot   ( 7146): ^_^_^_^_^ resize 2279704, 320, 480
I/godot   ( 7146): editor: 0
D/PhoneWindow( 7146): couldn't save which view has focus because the focused view com.android.godot.GodotView@406b9008 has no id.
D/SensorManager( 7146): unregisterListener:: disable all sensors for this listener,  listener = com.android.godot.Godot@4068af10
```

Note these lines
```
...
W/GodotView( 7146): creating OpenGL ES 1.1 context :
I/godot   ( 7146): ^_^_^_^_^ newcontext 2279704
I/godot   ( 7146): ^_^_^_^_^ resize 2279704, 480, 320
...
I/godot   ( 7146): ^_^_^_^_^ resize 2279704, 320, 480
...
```

Please forgive my English, it's the best I got with google translator xD

G.M.

**Comment by akien-mga** [comment]
Is this issue still valid in the current master branch?

**Comment by akien-mga** [comment]
Ping :)

**Comment by mohaalak** [comment]
@akien-mga I think it was corrected even in 1.1 I released two games in portrait  mode

**Comment by akien-mga** [comment]
Thanks, closing then. If the issue can still be reproduced in some settings, we'll get a new report :)
